backend/src/main/java/gov/cdc/usds/simplereport/db/model/auxiliary/ResultUploadErrorType.java

```diff
@@ -4,4 +4,5 @@ public enum ResultUploadErrorType {
   MISSING_HEADER,
   INVALID_DATA,
   MISSING_DATA,
+  INCLUDES_NON_AVAILABLE_DISEASE
```

Comment:
nit: I'm wondering if this can be something more concise like `UNAVAILABLE_DISEASE`? 🤔